.github/workflows/site.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     if: ${{ github.event_name == 'pull_request' }}
     steps:
     - name: Git Checkout
-      uses: actions/checkout@v4.0.0
+      uses: actions/checkout@v3.3.0
```

Comment:
Is this intentional?

Reply:
Probably because of zio-sbt?

Reply:
Yes.

Reply:
This issue can be fixed when having the latest pre-release version of zio-sbt. I'll make a release as soon as possible.